migrations/m150623_212711_fix_username_notnull.php

```diff
@@ -36,7 +36,7 @@ public function down()
             if ($this->dbType == 'sqlsrv') {
                 $this->dropIndex('{{%user_unique_username}}', '{{%user}}');
             }
-            $this->alterColumn('{{%user}}', 'username', $this->string(255)->null());
+            $this->alterColumn('{{%user}}', 'username', $this->string(255), ' NULL');
```

Comment:
Never mind, there was a very ugly bug in the check of the db type. (though for some strange reason in my test environment migration with mariadb worked without errors).